.cirrus.yml

```
@@ -0,0 +1,9 @@
+task:
```

Maybe name the task?

```suggestion
Test_task:
```

Can also be achieved by adding `name: Test` under `task`.